src/extension.ts

```diff
@@ -20,6 +20,22 @@ import * as vscode from 'vscode';
 import * as fs from 'fs';
 import * as path from 'path';
 
+// tslint:disable-next-line:no-var-requires
+const iotjs = require('/media/kutus/iotjs-vscode-extension/src/IotjsFunctions.json');
```

Comment:
This path a bit strange. Is that valid?

Reply:
You are right! That should not be hardcoded. Nice catch, i've missed that.